package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@atmedia/react-gpt",
-    "version": "2.1.6",
+    "version": "2.1.7",
```

Comment:
I just bumped this because 🤷 If we're being semantic, it's probably a major version change.